src/brstm/tests/assets/info.md

```diff
@@ -1,3 +1,4 @@
 # Credits
 
 - "Malfidus_news_loop.brstm" is created using BrawlBox using [Malfidus_news_loop.ogg](https://commons.wikimedia.org/wiki/File:Malfidus_news_loop.ogg), which is relased under Public Domain by User:Malfidus
+- "little_endian.brstm" is contributed by @ic-scm in https://github.com/kenrick95/nikku/issues/17#issuecomment-697368221

```

Comment:
@ic-scm I hope you don't mind me putting your "le-1.brstm" as part of the test suite ✌️